portfolio/src/main/webapp/index.html

```diff
@@ -2,57 +2,85 @@
-            <ul class="nav_elements"><a href="#experience">EXPERIENCE</ul>
+            <ul class="nav_elements"><a href="#skills">SKILLS</ul>
             <ul class="nav_elements"><a href="#projects">PROJECTS</a></ul>
-            <ul class="nav_elements"><a href="#education">EDUCATION</a></ul>
+            <ul class="nav_elements"><a href="#experience">EXPERIENCE</a></ul>
             <ul class="nav_elements"><a href="resume.pdf">RESUME</ul>
             <ul class="nav_elements"><a href="#contact">CONTACT</a></ul>
-        </nav>
-        <div class="container">
-            <div>
-                <h1>ABOUT ME</h1>
-                <h3>Hey, I am Palak Mittal from Rajasthan, India. jdecbldukbc;ejdwlM NXUCNKAKHEUIHCNBHD YCUJBVGDYJNBCG UKNBHUJNBJNBG UJHUJNBYU JNBGYUJN  HJ</h3>
+        </nav> 
+        <div class="cards"><span class="quote">W e&nbsp&nbsp&nbsp&nbspn e e d&nbsp&nbsp&nbsp&nbspm o r e&nbsp&nbsp&nbsp&nbspd o e r s ,&nbsp&nbsp&nbsp&nbspl e s s&nbsp&nbsp&nbsp&nbspt a l k e r s</span>
+        </div> 
+        <div class="cards">
+            <div class="about-text" style="color:white;">
+                <h1>Hi, I'm Palak</h1>
+                <hr style="float:left;"><br>
+                <h3>I build apps and coder by passion who is always ready to learn a new skill.<br> I love to have a new challenge as an apetite.</h3>
             </div>
             <div>
                 <img src="profile_image.jpg" class="img"></img>
             </div>
         </div>
     </header>
-    <div class="container2" id="experience">
+    <div class="container2" id="skills">
         <div class="container">
-            <h1>EXPERIENCE</h1>
-            <h3>Hey, I am Palak Mittal from Rajasthan, India. jdecbldukbc;ejdwlM NXUCNKAKHEUIHCNBHD YCUJBVGDYJNBCG UKNBHUJNBJNBG UJHUJNBYU JNBGYUJN  HJ</h3>
-            <span class="dot dot-blue"></span>
+            <h1>SKILLS & EDUCATION</h1>
+            <h3>Currently, I am an undergrad in Information Technology at IIIT Allahabad.<br><br>By far I gained a small bucket of skillset.<br>For competitive coding, I work with C++ most of times and development purpose made me to explore Java on Android Studio, Linux and Git.<br>I also have some hands on data manipulation DBMS(PL/SQL) and React JS.<h3>
             <span class="dot dot-blue"></span>
+            <span class="dot dot-red"></span>
         </div>
     </div>
     <div class="container1" id="projects">
         <div class="container">
             <h1>PROJECTS</h1>
-            <h3>Hey, I am Palak Mittal from Rajasthan, India. jdecbldukbc;ejdwlM NXUCNKAKHEUIHCNBHD YCUJBVGDYJNBCG UKNBHUJNBJNBG UJHUJNBYU JNBGYUJN  HJ</h3>
-            <span class="dot dot-white"></span>
+            <h3>Here is some of the stuff developed by me as a beginner till today.</h3>
+        </div>
+       <div class="cards">
+            <div class="card" >
+                <a href="https://github.com/thesmallstar/Saarthi">
```

Comment:
nit: might be a good idea to add target="_blank" so that this link opens in a new window. Same for the links below.

(This change is optional. It might be difficult for people to navigate back to the website if the external links are opened in the same tab)